tests/scenario_tests/test_attachment_actions.py

```diff
@@ -105,7 +105,7 @@ def test_failure_without_type(self):
         app.action("unknown")(simple_listener)
         response = app.dispatch(request)
         assert response.status == 404
-        assert self.mock_received_requests["/auth.test"] == 2
+        assert self.mock_received_requests["/auth.test"] == 1
```

Comment:
Now that the `auth.test` API call result is cached, the call count must be always 1 for the same token.